tests/perf/http_client_perf.cc

```diff
@@ -0,0 +1,143 @@
+/*
+ * This file is open source software, licensed to you under the terms
+ * of the Apache License, Version 2.0 (the "License").  See the NOTICE file
+ * distributed with this work for additional information regarding copyright
+ * ownership.  You may not use this file except in compliance with the License.
+ *
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+/*
+ * Copyright (C) 2024 ScyllaDB
+ */
+
+#include <seastar/core/seastar.hh>
```

Comment:
Would have been really nice to have a comment at the top of the file saying what this file tests. Maybe even cut-and-paste from your commit message.

Please also explain somewhere in the code (at the top of the file, or closer to where this happens in the code), exactly which sort of workload is being measured. If I understand the code correctly, you create a single connection and then repeatedly send one request and wait for one response - concurrency isn't being exercised, and it is the latency - not maximum throughput - which is actually being measured. I'm not saying it's necessarily bad, but you (and other developers who might want to use this benchmark) should know what is really being measured.

Reply:
Added